main.py

```diff
@@ -79,6 +79,16 @@ async def IsMemberWorking(ctx, discorduser, date = datetime.today().strftime('%d
 
     await ctx.send(content = "Done", delete_after = 0.1)
 
+@slash.slash(name = "GetEndOfMonthReport", description = "Calculates the salary deduction percentage for the provided month", options = UI.CreateGetEndOfMonthReportOptions(), guild_ids = guild_ids)
+async def GetEndOfMonthReport(ctx, month = None, year = None):
+    if Utilities.IsAdmin(ctx.author):
+        embed = UI.CreateGetEndOfMonthReportEmbed(member_db.GetMembers(), month, year)
```

Comment:
this command should take in members, then default to all

Reply:
Done